### ISSUE TITLE
MariaDB Add-On: Disk usage (huge amounts written)

### ISSUE THREAD

**Issue by derhappy** [home-assistant/addons#2267]
### Describe the issue you are experiencing

I coincidental found that the MariaDB add-on wrote (and read) huge amounts of data to and from the disk.
I's been about 200G written, 66G read in 1 1/2 days (since restart).
I'm not sure when this started, since I just recently started monitoring my system. According to the disk usage history, it must have been around mid / end of October. It's hard to tell since I started using frigate at that time and it initially messed with the disk a lot. That's also the only reason I even stumbled upon it.
After mounting a HDD to /media/frigate in order to take the stress from the main SSD I expected the writes to the SSD to drop significantly. After some days I just ran iostat and was shocked to see the amounts written in such a short time. Afterwards I ran iotop for some hours finding one of the MariaDB tasks to be the villain.
Our DB is around 10G in total, and not growing noticeably, so it's not the sensor data itself. After disabling frigate along with some other integrations and sensors which produce huge amounts of data, I did not see any drop in disk usage. Restarts and alike also did not change anything.
After some days I finally found the time to replace the DB, and then the writes finally dropped.

I have no idea why this was happening, nor any logs. It might be the combination of a rather big DB in combination with the performance optimizations for smaller systems like Raspberry. Or some settings that are fine on a Raspberry / ARM, but lead to issues on an AMD64 system. 
Just wanted this to be documented for others to be found and be aware. Since the data does not seem to get more the disk never gets full but might finally fail without one ever knowing why.

Maybe if others have the same issue we might figure out the cause. If I'm the only one, that's even better.

TL/DR
MariaDB add-on in a supervised set up on an Intel i3 with Ubuntu wrote huge amounts of data without filling up the disk (over 5G per hour).
Moving to the latest official MariaDB docker container fixed the issue. The writes are down (less than 0,5G per hour).

### What type of installation are you running?

Home Assistant Supervised

### Which operating system are you running on?

Other (e.g., Raspbian/Raspberry Pi OS/Fedora)

### Which add-on are you reporting an issue with?

MariaDB

### What is the version of the add-on?

2.4.0

### Steps to reproduce the issue

1.
2.
3.
...

### Anything in the Supervisor logs that might be useful for us?

```txt
nothing
```

### Anything in the add-on logs that might be useful for us?

```txt
nothing suspicious
```
